pkg/cmd/codespace/ssh.go

```diff
@@ -125,7 +132,7 @@ func (a *App) SSH(ctx context.Context, sshArgs []string, opts sshOptions) (err e
 	startSSHOptions := liveshare.StartSSHServerOptions{}
 
 	if shouldGenerateSSHKeys(args, opts) && sshContext.HasKeygen() {
-		keyPair, err := sshContext.GenerateSSHKey("codespaces", "")
+		keyPair, err := sshContext.GenerateSSHKey(automaticPrivateKeyName, "")
```

Comment:
Yeah, @josebalius also pointed this out. Added backwards compatibility such that if the old public and old private keys both exist then we rename them. Otherwise we leave any existing files alone and create the new keys. In all cases we no longer use the old keys anymore.